modules/javafx.controls/src/test/java/test/javafx/scene/control/ListCellTest.java

```diff
@@ -1106,6 +1106,30 @@ public void testMisbehavingCancelEditTerminatesEdit() {
         }
     }
 
+    /**
+     * See also: <a href="https://bugs.openjdk.org/browse/JDK-8187314">JDK-8187314</a>.
+     */
+    @Test
+    public void testEditCommitValueChangeIsReflectedInCell() {
```

Comment:
this test passes with the fix, fails without as expected.
but for some reason I cannot reproduce the issue with ListView in the MonkeyTester
https://github.com/andy-goryachev-oracle/MonkeyTest

(List View Page -> select 'editable' -> double click to edit, type "updated" to save changes, any other text to reject)